transport/can/can-buffer.c

```diff
@@ -23,12 +23,14 @@
 
 void AsebaSendBuffer(AsebaVMState *vm, const uint8* data, uint16 length)
 {
+	(void) vm;
```

Comment:
I guess this aims at removing a "variable not used" warning. I was not aware about this usage before this pull request, do you have a link to a reference/rationale for it? I'm curious!